typings/index.d.ts

```diff
@@ -2436,8 +2449,7 @@ export interface TextBasedChannelFields extends PartialTextBasedChannelFields {
     options?: InteractionCollectorOptions<T>,
   ): InteractionCollector<T>;
   createMessageCollector(options?: MessageCollectorOptions): MessageCollector;
-  startTyping(count?: number): Promise<void>;
-  stopTyping(force?: boolean): void;
+  sendTyping(count?: number): Promise<void>;
```

Comment:
The `count` parameter doesn't exist anymore.

```suggestion
  sendTyping(): Promise<void>;
```